.github/ISSUE_TEMPLATE/new_release_repository.md

```diff
@@ -7,8 +7,18 @@ title: 'Add new release repositories'
 * Name of the release team: 
 
 * Release repositories to add:
+  <!--
+  Before release repositories are created, repositories and packages should be submitted to the official rosdistro repository https://github.com/ros/rosdistro for naming review.
+  The ros/rosdistro source entry link must be included for each repository.
+  Guidelines for package naming are describe in REP-144: https://www.ros.org/reps/rep-0144.html
+
+  If there are any existing release repositories which should have their contents imported into the ros2-gbp organization, list and link to them here.
+  Leave the checkbox _unchecked_. The ros2-gbp administrator will check the box when they have completed the repository import.
+  -->
   * [FIRST_REPOSITORY](SOURCE URL)
+    * [ros/rosdistro source entry](LINK TO SOURCE FIELD IN ROS/ROSDISTRO)
     * [ ] There is an existing release repository which should be imported: <RELEASE REPOSITORY URL>
   * [SECOND REPOSITORY](SOURCE URL)
```

Comment:
How about making this 2nd repo entry option, or simply delete? This made me go through

"why 2nd repo is asked here...Hm, maybe there are people who want to submit a single request for multiple repos, but I'm not sure how common that would be?",

and finally I just made a single entry :sweat_smile:

Reply:
Thanks for the feedback, I did assume that many "new" requests for ros2-gbp repositories would be in batches of two or more, but that probably only made sense when the ros2-gbp org project was new itself and adding repositories to an existing team probably is a one-at-a-time deal except in cases where a new suite of packages in separate repositories is being released for the first time.

My general rule of thumb for templates is that you should show how to sequence things that can be duplicated and I would much rather one issue with two repositories than one issue per repository.